UKHO.MaritimeSafetyInformation.Web/UKHO.MaritimeSafetyInformation.Web/Views/NoticestoMariners/Resellers.cshtml

```diff
@@ -0,0 +1,203 @@
+                <thead>
+                <tr>
+                    <th width="65%">Value Added Reseller</th>
+                    <th>Website</th>
+                </tr>
+                </thead>
+                <tbody>
+                <tr>
+                    <td>American Nautical Services</td>
+					<td><a href="https://www.amnautical.com" target="_blank">www.amnautical.com</a></td>
+                </tr>
+                <tr>
+                    <td>B Cooke & Son Ltd</td>
+					<td><a href="https://www.bcookeandson.co.uk" target="_blank">www.bcookeandson.co.uk</a></td>
+                </tr>
+                <tr>
+                    <td>Boat Books Australia Pty Ltd</td>
+					<td><a href="https://www.boatbooks-aust.com.au" target="_blank">www.boatbooks-aust.com.au</a></td>
+                </tr>
+                <tr>
+                    <td>Bogerd Martin (Hong Kong) Ltd</td>
+					<td><a href="https://www.martin.be" target="_blank">www.martin.be</a></td>
+                </tr>
+                <tr>
+                    <td>C & C Marine Combine</td>
+					<td><a href="https://www.cc-marine.com/" target="_blank">www.cc-marine.com</a></td>
+                </tr>
+                <tr>
+                    <td>CAIM (Cooperativa per Azioni Armamento Imprese Maritime)</td>
+					<td><a href="https://www.caim.it" target="_blank">www.caim.it</a></td>
+                </tr>
+                <tr>
+                    <td>Chart Track</td>
+					<td><a href="https://www.charttrack.com" target="_blank">www.charttrack.com</a></td>
+                </tr>
+                <tr>
+                    <td>ChartWorld International Limited</td>
+					<td><a href="https://www.chartworld.com" target="_blank">www.chartworld.com</a></td>
+                </tr>
+                <tr>
+                    <td>Chersoft Ltd</td>
+					<td><a href="https://www.chersoft.com" target="_blank">www.chersoft.com</a></td>
+                </tr>
+                <tr>
+                    <td>Da Gama Maritime Ltd</td>
+					<td><a href="https://www.dgmaritime.com" target="_blank">www.dgmaritime.com</a></td>
+                </tr>
+                <tr>
+                    <td>Datema Delfzijl BV</td>
+					<td><a href="https://www.datema.nl" target="_blank">www.datema.nl</a></td>
+                </tr>
+                <tr>
+                    <td>David Store Navigation Management</td>
+					<td><a href="https://www.dsnmltd.com" target="_blank">www.dsnmltd.com </a></td>
+                </tr>
+                <tr>
+                    <td>E W Liner Charts & Publications</td>
+					<td><a href="https://www.ewliner.com" target="_blank">www.ewliner.com</a></td>
+                </tr>
+                <tr>
+                    <td>Easy Marine</td>
+					<td><a href="https://www.easymarine.it" target="_blank">www.easymarine.it</a></td>
+                </tr>
+                <tr>
+                    <td>Guangzhou Haining Maritime Service</td>
+					<td>---</td>
```

Comment:
Existing MSI application displaying N/A for website for this VAR. Not sure if we keep it same?
```suggestion
					<td>N/A</td>
```

Reply:
Okay, I preferred `---` but not too worried either way.